pupil-spa/src/app/familiarisation-area/familiarisation-area.component.spec.ts

```diff
@@ -46,19 +60,52 @@ describe('FamiliarisationAreaComponent', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should redirect to colour contrast when enabled', () => {
+  it('should redirect to colour contrast when enabled', async () => {
     spyOn(mockQuestionService, 'getConfig').and.returnValue({ colourContrast: true });
-    component.onClick();
+    await component.onClick();
     fixture.whenStable().then(() => {
       expect(mockRouter.navigate).toHaveBeenCalledWith(['colour-choice']);
     });
   });
 
-  it('should redirect to the settings page when colour contrast not enabled', () => {
-    spyOn(mockQuestionService, 'getConfig').and.returnValue({ colourContrast: false });
-    component.onClick();
+  it('should redirect to the settings page when colour contrast not enabled', async () => {
+    spyOn(mockQuestionService, 'getConfig').and.returnValue({colourContrast: false});
+    await component.onClick();
     fixture.whenStable().then(() => {
       expect(mockRouter.navigate).toHaveBeenCalledWith(['access-settings']);
     });
   });
+  describe('When featureUseHpa is true', () => {
```

Comment:
what if HPA mode is not enabled.  Is this covered?

Reply:
@GuyHarwood covered now as discussed